adyenv6core/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -630,18 +627,17 @@ public OrderData handle3DS2Response(final HttpServletRequest request) throws Exc
 
         PaymentsResponse.ResultCodeEnum resultCode = paymentsResponse.getResultCode();
 
-        if (PaymentsResponse.ResultCodeEnum.AUTHORISED == resultCode) {
+        if (resultCode != PaymentsResponse.ResultCodeEnum.IDENTIFYSHOPPER && resultCode != PaymentsResponse.ResultCodeEnum.CHALLENGESHOPPER) {
+            //payment is finished
```

Comment:
payment is not finished. comment is misleading.

Reply:
will remove